auth0/src/test/java/com/auth0/android/provider/WebAuthProviderTest.java

```diff
@@ -1550,6 +1584,12 @@ private String createHash(@Nullable String idToken, @Nullable String accessToken
             }
             hash = hash.concat("error=" + error);
         }
+        if (errorDescription != null) {
+            if (!hash.endsWith("&")) {
```

Comment:
I guess i wanted to do that check but got lost and used the same `&` for comparison. Good catch 👍